assets/js/blocks/active-filters/block.js

```diff
@@ -117,19 +125,34 @@ const ActiveFiltersBlock = ( {
 	}
 
 	const TagName = `h${ blockAttributes.headingLevel }`;
-	const listClasses = classnames( 'wc-block-active-filters__list', {
-		'wc-block-active-filters__list--chips':
-			blockAttributes.displayStyle === 'chips',
-	} );
+	const listClasses = classnames(
+		'wc-block-active-filters__list',
+		textColorClass,
+		{
+			'wc-block-active-filters__list--chips':
+				blockAttributes.displayStyle === 'chips',
+		}
+	);
 
 	return (
 		<>
 			{ ! isEditor && blockAttributes.heading && (
-				<TagName className="wc-block-active-filters__title">
+				<TagName
+					className={ classnames(
+						'wc-block-active-filters__title',
+						textColorClass
+					) }
+					style={ textColorStyle }
+				>
 					{ blockAttributes.heading }
 				</TagName>
 			) }
-			<div className="wc-block-active-filters">
+			<div
+				className={ classnames(
+					'wc-block-active-filters',
+					textColorClass
```

Comment:
Same here, if we remove this class, would it just work? Color will be inherited from the parent div wrapper.